Aplib.Core/Desire/Goal.cs

```diff
@@ -0,0 +1,79 @@
+namespace Aplib.Core.Desire
+{
+    public class Goal
+    {
+        /// <summary>
+        /// The goal is considered to be completed, when the distance of the
+        /// <see cref="CurrentHeuristics"/> is below this value.
+        /// </summary>
+        private const float Epsilon = 0.005f;
+
+        /// <summary>
+        /// The <see cref="Heuristics"/> of the current state of the game.
```

Comment:
Documentation for properties should always begin with `gets or sets` (depending on what is available).

Example:
```suggestion
        /// Gets the <see cref="Heuristics"/> of the current state of the game.
```